lib/broadway_ecto_job/producer.ex

```diff
@@ -64,16 +64,23 @@ defmodule BroadwayEctoJob.Producer do
   def mark_in_progress(message) do
     config = message.metadata[:config]
 
-    {:ok, updated_job} =
+    result =
```

Comment:
`mark_in_progress` is expected to always return a message, so it will still fail somewhere later... Let me think how to fix this.

Reply:
@iurimadeira ping :)